test/loader-common.js

```diff
@@ -27,8 +27,22 @@ export const LoaderCommonHooks = {
     };
     this.seeking = false;
     this.hasPlayed = true;
+    this.paused = false;
+    this.playbackRate = 1;
     this.fakeHls = {
-      xhr: xhrFactory()
+      xhr: xhrFactory(),
+      tech_: {
+        paused: () => this.paused,
+        playbackRate: () => this.playbackRate
+      }
+    };
+    this.goalBufferLength = () => {
```

Comment:
This is the function I mentioned before that you may be able to avoid duplicating here.